frame/transaction-payment/asset-tx-payment/src/payment.rs

```diff
@@ -126,8 +126,15 @@ where
 		if !matches!(can_withdraw, WithdrawConsequence::Success) {
 			return Err(InvalidTransaction::Payment.into())
 		}
-		<T::Fungibles as Balanced<T::AccountId>>::withdraw(asset_id, who, converted_fee)
-			.map_err(|_| TransactionValidityError::from(InvalidTransaction::Payment))
+		<T::Fungibles as Balanced<T::AccountId>>::withdraw(
```

Comment:
just an observation: the list of `withdraw` params has grown to a point that it is hard to follow, at least the boolean flags.

Reply:
Fixed.